.github/workflows/pr-command.yml

```diff
@@ -0,0 +1,90 @@
+        uses: actions/checkout@v4
+      # Using an app is the recommended way to allow recursive operations.
+      # Apps offer a way to have reduced scoped permissions (like GITHUB_TOKEN)
+      # and short lifespan, unlike PAT.
+      # As we are trialing this approach tho, it is fine to play with a PAT and not spend the
+      # extra time setting up an app
+      #
+      # - name: Generate App Token
+      #   id: generate-token-key-pair
+      #   if: steps.check-maintainer.outputs.result == 'true'
+      #   uses: actions/create-github-app-token@v1
+      #   with:
+      #     private-key: ${{ secrets.MIXXX_BOT_APP_PRIVATE_KEY }}
+      #     app-id: ${{ vars.MIXXX_BOT_APP_ID }}
+      - name: Softfix
+        if: steps.check-maintainer.outputs.result == 'true'
+        uses: daschuer/softfix@v3
+        env:
+          # PAT setup with content:write and pull_request:write
+          repo-token: ${{ secrets.MIXXX_SOFTFIX_PAT }}
+
+  skip-ci:
+    name: Skip ci action
+    if: github.event.issue.pull_request != '' && (startsWith(github.event.comment.body, '/enable-ci') || startsWith(github.event.comment.body, '/disable-ci'))
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check if commenter is maintainer
+        id: check-maintainer
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const response = await github.rest.repos.getCollaboratorPermissionLevel({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              username: context.payload.comment.user.login
+            });
+
+            const isMaintainer = ['admin', 'write'].includes(response.data.permission);
+            return isMaintainer;
+      - name: Checkout repository
+        if: steps.check-maintainer.outputs.result == 'true'
+        uses: actions/checkout@v4
+      # Using an app is the recommended way to allow recursive operations.
+      # Apps offer a way to have reduced scoped permissions (like GITHUB_TOKEN)
+      # and short lifespan, unlike PAT.
+      # As we are trialing this approach tho, it is fine to play with a PAT and not spend the
+      # extra time setting up an app
+      #
+      # - name: Generate App Token
+      #   id: generate-token-key-pair
+      #   if: steps.check-maintainer.outputs.result == 'true'
+      #   uses: actions/create-github-app-token@v1
+      #   with:
+      #     private-key: ${{ secrets.MIXXX_BOT_APP_PRIVATE_KEY }}
+      #     app-id: ${{ vars.MIXXX_BOT_APP_ID }}
+      - name: Add or remove the skip-ci label
+        run: >-
+          gh pr edit
+            ${{ startsWith(github.event.comment.body, '/disable-ci') && '--add-label' || '--remove-label' }} 'skip-ci'
+            ${{ github.event.issue.pull_request }}
+        if: steps.check-maintainer.outputs.result == 'true'
+        env:
+          # PAT setup with pull_request:write
+          GH_TOKEN: ${{ secrets.MIXXX_PR_COMMAND_PAT }}
```

Comment:
PAT to be created for @mixxxbot and stored as secret

Reply:
That would be `github_pat_anVzdCBhIHByYW5rIGJybw==` ;)